docs/src/tutorials/tutorial_1.md

```diff
@@ -2,6 +2,350 @@
+> It has a [great tutorial](https://juliagraphics.github.io/Luxor.jl/stable/tutorial/) that will give you an even greater understanding of how `Javis.jl` works.
+
+## The `javis` Function - the Heart of `Javis.jl`
+
+In order to use `Javis`, we will start with the following import:
+
+```julia
+using Javis
+
+```
+
+Then, we will introduce our first and most import function from `Javis` - the `javis` function:
+
+```
+javis(
+    video = [[video]], 
+    actions = [[actions]], 
+    pathname = [[pathname]]
+)
+```
+
+The `javis` function has three key word parameters that the user must define in order to create an animation:
+
+- `video` - a `Video` struct
+- `actions` - a vector of `Action` objects
+- `pathname` - a `String` 
+
+These kwargs are further explained in the following sections. 
+
+### `video`
+
+The `video` argument requires a `Video` struct.
+This `Video` struct creates the canvas upon which we will draw. 🎨 
+To create a `Video` object, one defines the constructor with a width and height:
+
+```julia
+myvideo = Video(500,500) # 500 x 500 // width x height
+
+```
+
+### `actions`
+
+Core to not only the `javis` function but the entire `Javis.jl` library are `Action` objects. 
+`Action` objects will be better explained in [Tutorial 2](tutorial_2.html).
+For now, just know that an `Action` object is what draws shapes or moves shapes on a `Video`. In the `javis` function, the `actions` kwarg is a vector of `Action` objects.
+
+### `pathname`
+
+The `pathname` is a `String` that defines where to put the output animation on your computer.
+In the `pathname`, you must end the path with the name of the animation and its associated filetype extension.
+Here is an example invocation of the `javis` function with a provided path name:
+
+```julia
+javis(
+    myvideo, 
+    myactions, 
+    pathname = "/home/myanimation.gif"
+)
+```
+
+## Making Our Animation
+
+We need to set-up a few functions that will be able to make our animation! 
+
+Our first function is the `ground` function.
+The `ground` function sets the background to white and the "pen" color to black:
+
+```julia
+function ground(args...) 
+    background("white") # canvas background
+    sethue("black") # pen color
+end
+```
+
+> **NOTE:** One may wonder, "why is `args...` needed in the `ground` function?"
+> To explain, each user-defined function gets three arguments `video`, `action`, and `frame` provided by `javis`
+> These arguments are defined below:
+>
+> - `video`: Video struct
+> - `action`: Action struct
+> - `frame`: the current frame number
+>
+> Although these arguments are irrelevant for the `ground` function, we need to write `args...` such that Julia actually knows that we have a method that accepts those three arguments.
+> The `...` basically stands for as many arguments as you want.
+
+As `Javis` does not provide a method for drawing circles, we need to make one ourselves!
```

Comment:
I would rephrase as Luxor provides the functionality to draw circles but we also want to specify the color of the circle here.